docs/tutorials/quantum_volume.rst

```diff
@@ -142,7 +142,10 @@ enhancements might be required (See Ref. [2] for details).
 
 .. jupyter-execute::
 
-    exps = [QuantumVolume(range(i), trials=200) for i in range(3, 6)]
+    exps = []
+    for i in range(3,6):
```

Comment:
Can you change this one back to a list comprehension?

Reply:
Thanks for the comments, updated~